src/Codeception/Lib/Generator/Actions.php

```diff
@@ -205,4 +206,26 @@ public function getNumMethods()
     {
         return $this->numMethods;
     }
+
+    private function createReturnTypeHint(\ReflectionMethod $refMethod)
+    {
+        if (PHP_VERSION_ID < 70000) {
+            return '';
+        }
+
+        $returnType = $refMethod->getReturnType();
+
+        if ($returnType === null) {
+            return '';
+        }
+
+        return strtr(
+            ': {nullable}{trailingBackslash}{type}',
+            [
+                '{nullable}' => $returnType->allowsNull() ? '?' : '',
+                '{trailingBackslash}' => $returnType->isBuiltin() ? '' : '\\',
```

Comment:
This backslash is leading, not trailing.

Also `strtr` syntax looks like overkill, simple string concatenation or sprinf(': %s%s%s', expressions) would be much more concise.

Reply:
True :)